bin/check-redis-list-length.rb

```diff
@@ -63,21 +63,45 @@ class RedisListLengthCheck < Sensu::Plugin::Check::CLI
          description: 'Redis list KEY to check',
          required: true
 
+  banner "Usage: #{File.basename($PROGRAM_NAME)} (options)\n" \
+    'Number of items in list key will have to be below threshold if warning is smaller than critical, and above otherwise'
+
   def run
     options = { host: config[:host], port: config[:port], db: config[:database] }
     options[:password] = config[:password] if config[:password]
-    redis = Redis.new(options)
 
+    check_length_is_above_threshold = false
+    if config[:warn] > config[:crit]
```

Comment:
We should handle them being equal. Ideally something along these lines:
```ruby
if config[:warn] > config[:crit]
  check_length_is_above_threshold = true
elsif config[:warn] < config[:crit]
  check_length_is_above_threshold = false
else
  unknown 'Unable to determine the appropriate threshold as warn and crit should never be the same'
end
```

And drop the assignment outside of the conditional. This handles all 3 scenarios and tells the user when they try to do something that is not within the current capabilities of the check.